app/pages/reservation/reservation-information/ReservationInformationForm.js

```diff
@@ -285,7 +285,7 @@ class UnconnectedReservationInformationForm extends Component {
               t('common.billingEmailAddressLabel'),
             )
           }
-          <h2 className="app-ReservationPage__title">{t('ReservationInformationForm.eventInformationTitle')}</h2>
+          <h2 className="app-renderTermsField">{t('ReservationInformationForm.eventInformationTitle')}</h2>
```

Comment:
Why did you change the className here?

Reply:
Mistakenly got changed. Reverted back in latest commit.